spec/views/rb_taskboards/show_spec.rb

```diff
@@ -117,7 +125,7 @@
 
       stories.each do |story|
         expect(rendered).to have_selector "#story_#{story.id}" do
-          with_selector ".subject", story.subject
+          with_selector '.subject', story.subject
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.